tap_linkedin_ads/__init__.py

```diff
@@ -29,23 +30,24 @@ def do_discover(client, config):
 
 @singer.utils.handle_top_exception(LOGGER)
 def main():
-
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
+    with LinkedinClient(parsed_args.config['client_id'],
+                        parsed_args.config['client_secret'],
+                        parsed_args.config['refresh_token'],
+                        REQUEST_TIMEOUT,
+                        parsed_args.config['user_agent'],
+                       ) as client:
 
-    with LinkedinClient(access_token=parsed_args.config['access_token'],
-                        user_agent=parsed_args.config['user_agent'],
-                        timeout_from_config=parsed_args.config.get('request_timeout')) as client:
         state = {}
         if parsed_args.state:
             state = parsed_args.state
-
+        config = parsed_args.config
```

Comment:
If we do this above L34, we could just used the config instead of `parsed_args.config` there as well.

Reply:
yes , i have same changes